### ISSUE TITLE
Custom Resources reference kube_customresource_ and it looks to have changed to kube_crd_

### ISSUE THREAD

**Issue by stewartshea** [kubernetes/kube-state-metrics#1970]
**What happened**:
While reading through the docs https://github.com/kubernetes/kube-state-metrics/blob/main/docs/customresourcestate-metrics.md I was searching and debugging why the custom resources weren't being displayed. It looks (at least in my instance) that the resources have shown up now as `kube_crd_` instead of `kube_customresource_`. 
 
**What you expected to happen**:
I was searching for `kube_customresource_` in our prom instance and unable to find the desired custom metrics. 

**How to reproduce it (as minimally and precisely as possible)**:
Basic setup with a custom resource config file as outlined here:  https://github.com/kubernetes/kube-state-metrics/blob/main/docs/customresourcestate-metrics.md

Example configuration: 
```
        - groupVersionKind:
            group: "runwhen.com"
            kind: "Location"
            version: "v1"
          metrics:
            - name: "heartbeat"
              help: "Latest Heartbeat"
              each:
                type: Gauge
                gauge:
                  path: [status]
                  valuesFrom: [heartbeat]
              labelsFromPath:
                locationType: [spec, locationType ]

```

Example result from the kube-state-metrics /metrics endpoint: 
```
kube_crd_heartbeat{group="runwhen.com",kind="Location",locationType="public",version="v1"} 1.675200259e+09
```

**Anything else we need to know?**:
I *think* this is just a doc update to change `customresource` to `crd` so that other readers can quickly find their custom resource metrics while implementing this solution. 

**Environment**:
- kube-state-metrics version: 2.7.0
- Kubernetes version (use `kubectl version`): 1.23.14-gke-401
- Cloud provider or hardware configuration: GKE
- Other info:

**Comment by mrueg** [comment]
Thanks for your report and sending the PR @stewartshea! 

We started renaming the experimental CustomResource metrics in the main branch from kube_crd to kube _customresource (change is in https://github.com/kubernetes/kube-state-metrics/commit/488c0bcf590e3992799557eb0db4ea99e8d407ca) which will be released in v2.8.0. So the change proposed in #1972 is not needed.

Please check https://github.com/kubernetes/kube-state-metrics/blob/release-2.7/docs/customresourcestate-metrics.md (release-2.7 branch) for the documentation on the current release.

Let me know if there's anything else and feel free to reopen the issue/PR in case I misunderstood your question.

**Comment by stewartshea** [comment]
Thanks Manuel!

--
Shea Stewart (he/him)
c:647-972-5191 ***@***.***
Book a meeting https://calendly.com/stewartshea
